src/lsp/cobol_indent/indenter.ml

```diff
@@ -62,22 +72,37 @@ let indenter ~source_format (str:string) (rdl:indent_record list) range =
   String.concat "\n" strl
 
 (*indent a range of file, with the default indent_config*)
-let indent_range' ~dialect ~source_format ~range ~filename ~contents =
+let indent_range ~dialect ~source_format ~range ~filename ~contents =
+  let src_format =
```

Comment:
Here you could just always start with `SFFixed` directly and not bother inspecting `source_format` at all.

Reply:
True. This was initially written before #70, and I didn't revisit the code.